ui-tests/tests/results.spec.ts

```diff
@@ -32,6 +33,12 @@ test.describe('Results', () => {
   });
 
   test.beforeEach(async ({ page }) => {
+    await page
+      .locator('body')
+      .evaluate(
+        (element, path) => (element.dataset['profilerDir'] = path),
```

Comment:
Need to modify page config ahead of page load or to make the results path runtime callback